pylate/server/server.py

```diff
@@ -0,0 +1,99 @@
+import argparse
+from typing import List
+
+import batched
+import uvicorn
+from fastapi import FastAPI, HTTPException
+from pydantic import BaseModel
+
+from pylate import models
+
+app = FastAPI()
+
+
+class EmbeddingRequest(BaseModel):
```

Comment:
Would be great to add some docstring here

Reply:
Done